image/user/db/settings_v2_002.sql

```diff
@@ -44,7 +44,5 @@ INSERT OR IGNORE INTO settings_tab (path, value) VALUES
     ('gs_current_timezone_name', ''),
     ('gs_current_timezone_rules', ''),
     ('\ServiceTime\\gs_automatic_date_and_time_is_on', '1'),
-    ('\ServiceEink\\display_inverted_mode', '0');
-
-
-
+    ('\ServiceEink\\display_inverted_mode', '0'),
+    ('gs_device_unique_id', '');
```

Comment:
@mjan-dev @pholat Looks like this addition is triggering some problems with settings after OS update. Do we have a plan for handling database updates?

Reply:
If it is only in sql file it should not have impact on the database. Key will be created if nonexistent.

Reply:
I'll retest it.

Reply:
Confirmed that writing to a newly added field in settings DB causes corruption of settings for a passcode (enabled flag and passcode value). The issue is observed after update to OS version with new field and after the filed was written to.

Reply:
If it's going to be investigated, please report the bug and add reference to it here.

Reply:
Created and linked a bug https://appnroll.atlassian.net/browse/MOS-236